applications/serial_lte_modem/src/nrf52_dfu/slip.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#pragma once
+
+#ifndef SLIP_H__
+#define SLIP_H__
+
+#include <zephyr.h>
+
+#define SLIP_SIZE_MAX 128
+
+#ifdef __cplusplus
+extern "C" {
+#endif  /* __cplusplus */
+
+
+#define	SLIP_END			0300
+#define	SLIP_ESC			0333
+#define	SLIP_ESC_END			0334
+#define	SLIP_ESC_ESC			0335
+
+void encode_slip(uint8_t *pDestData, uint32_t *pDestSize,
```

Comment:
Old names are still used in the header.

Reply:
I changed lots of files and some of them are not saved when committing, and they are still "updated" status in my editor 😓 no mistake this round.